pyproject.toml

```diff
@@ -26,13 +26,13 @@ classifiers = [
     "Topic :: Scientific/Engineering",
     "Development Status :: 3 - Alpha"
 ]
-requires-python = ">=3.8,<3.11"
+requires-python = ">=3.9, <3.10"
 dependencies = [
     'pyobjc-core>=7.1; platform_system == "Darwin"',
     'darkdetect>=0.3.1',
     'pyobjc-framework-cocoa>=7.1; platform_system == "Darwin"',
-    'matplotlib==3.4.2',
-    'EasyReflectometryLib @ git+https://github.com/easyScience/EasyReflectometryLib.git@develop',
+    'matplotlib>=3.4.2',
+    'EasyReflectometryLib>=0.0.3',
     'easyApp @ git+https://github.com/easyScience/easyApp.git@develop',
```

Comment:
should we then merge `develop` into `main` in EasyApp? Otherwise, pypi will complain

Reply:
To my understanding the pipeline for the App only pushes artifacts to either a FTP server (MacOS, Windows) or Snapcraft (Linux) and we don't use PyPI for this repo